### DISCUSSION TITLE
aid ! how to get the authorization token

### DISCUSSION THREAD

**Discussion by drakewebs1** [masterking32/MasterHamsterKombatBot#35] [Q&A] [answered]
Hello, how can I get the token through the Telegram website since the hamster url has now been updated. I hope for help, please.

**Comment by masterking32** [comment, marked as answer]
https://youtu.be/cjWE7DmMFgw?feature=shared

**Comment by drakewebs1** [comment]
What happens is that I don't know if the scripts you put inside the extension help open the game from Telegram? Since I do everything you do in your video, I enter the telegram websites but when I press f12 and open the game, it doesn't let me enter, the QR image appears, I update but it doesn't work.

**Comment by masterking32** [comment]
8 hours ago we updated the codes of https://github.com/masterking32/MasterHamsterKombatBot/tree/main/useful_files did you test with the last versions?
